mw-webapp/src/service/DayReportService.ts

```diff
@@ -1,10 +1,49 @@
-import {collection, getDocs} from "firebase/firestore";
+import {collection, doc, getDoc, getDocs, setDoc, updateDoc} from "firebase/firestore";
 import {db} from "src/firebase";
 import {DayReportDTO} from "src/model/firebaseCollection/DayReportDTO";
+import {documentSnapshotToDTOConverter} from "src/service/converter/documentSnapshotToDTOConverter";
 import {querySnapshotToDTOConverter} from "src/service/converter/querySnapshotToDTOConverter";
 
 const PATH_TO_DAY_REPORTS_COLLECTION = "dayReports";
 
+/**
+ * New day report props without uuid for {@link data} for possibility to auto-generate uuid on firestore
```

Comment:
Use Omit instead of long comment if you should want specify type

Reply:
It's great!

Reply:
Done